src/plugins/intel_gpu/src/kernel_selector/core/actual_kernels/softmax/softmax_kernel_blocked.cpp

```diff
@@ -0,0 +1,139 @@
+// Copyright (C) 2022 Intel Corporation
```

Comment:
Can I ask why you separated this implementation from ref kernel?

Reply:
Existing ref kernel implemented in such way that it would not be enough to just replace offsets calculation with *_GET_INDEX macros to support blocked formats. 
Besides that, I have gut feeling that _GET_INDEX adds some performance overhead in this case, so I believe it makes sense to keep kernel which is optimized for plane formats.